pkg/operator/ceph/disruption/machinedisruption/reconcile.go

```diff
@@ -108,13 +118,11 @@ func (r *MachineDisruptionReconciler) Reconcile(request reconcile.Request) (reco
 		}
 		err = r.client.Create(context.TODO(), newMDB)
 		if err != nil {
-			logger.Errorf("failed to create mdb %+v", err)
-			return reconcile.Result{}, err
+			return reconcile.Result{}, fmt.Errorf("failed to create mdb %s: %+v", mdb.GetName(), err)
 		}
 		return reconcile.Result{}, nil
 	} else if err != nil {
-		logger.Errorf("%+v", err)
-		return reconcile.Result{}, err
+		return reconcile.Result{}, fmt.Errorf("%+v", err)
```

Comment:
I missed this earlier just return `err` here.

Reply:
done